.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 1.8, 1.11 ]
+    name: Test with Java ${{ matrix.java }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK ${{ matrix.java }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
```

The `hashFiles` should also include the `serverlibs.pom` (either as `hashFiles('**/pom.xml', '**/serverlibs.pom')` or `hashFiles('**/pom.xml', '**/*.pom')`), until it's eventually renamed…

Not sure if the `${{ matrix.java }}` should be part of the key too; technically the dependencies could be different due to Maven profiles. Worst case will be one build is cached and the other one needs to download the few dependencies that differ, but still downloads them every time it runs!
```suggestion
          key: ${{ runner.os }}-java-${{ matrix.java }}-m2-${{ hashFiles('**/pom.xml', '**/*.pom') }}
```

---

Good catch with the matrix key! I will include `'**/*.pom` as suggested and remove it once `serverlibs` is renamed